core/foundation/inc/RVersion.h

```diff
@@ -1,23 +1,8 @@
 #ifndef ROOT_RVersion
 #define ROOT_RVersion
 
-/* Version information automatically generated by installer. */
+#include "ROOT/RVersion.hxx"
 
-/*
- * These macros can be used in the following way:
- *
- *    #if ROOT_VERSION_CODE >= ROOT_VERSION(6,32,4)
- *       #include <newheader.h>
- *    #else
- *       #include <oldheader.h>
- *    #endif
- *
-*/
-
-#define ROOT_RELEASE "6.29/01"
-#define ROOT_RELEASE_DATE "Dec 21 2022"
-#define ROOT_RELEASE_TIME "11:50:22"
-#define ROOT_VERSION(a,b,c) (((a) << 16) + ((b) << 8) + (c))
-#define ROOT_VERSION_CODE ROOT_VERSION(6,29,1) /* 400641 */
+#define ROOT_RELEASE_TIME "00:00:00" /* not updated anymore */
```

Comment:
Why is it no longer updated?  Should it be somehow marked as deprecated?

Reply:
- I don't know how to mark it deprecated
- I don't think anyone cares about it not being "00:00:00"
- It's preprocessor stuff so I'd rather leave it alone :-)
-